reports/Wisconsin.md

```diff
@@ -31,18 +31,46 @@ id: wi-madison02
+tags: knee-on-neck, arrest, bike, protestor
 
 id: wi-milwaukee-1
 
 **Links**
 
 * http://www.milwaukeeindependent.com/articles/milwaukee-common-council-calls-investigation-use-knee-police-restrain-protestor/
-* https://old.reddit.com/r/gifs/comments/gxptyo/june_2_2020_milwaukee_police_punch_and_knee/
-* https://giphy.com/gifs/protest-milwaukee-blm-IzjGA5w04kTJTeP7Dz/fullscreen
-* [Location via Google Street View](https://www.google.ca/maps/place/N+6th+St+%26+W+Vliet+St,+Milwaukee,+WI+53205,+USA/@43.0483922,-87.9188126,3a,75y,1.17h,90t/data=!3m7!1e1!3m5!1se3cNwZvfdQD1wuznzZH5ow!2e0!6s%2F%2Fgeo2.ggpht.com%2Fcbk%3Fpanoid%3De3cNwZvfdQD1wuznzZH5ow%26output%3Dthumbnail%26cb_client%3Dsearch.gws-prod.gps%26thumb%3D2%26w%3D86%26h%3D86%26yaw%3D1.1730963%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192!4m5!3m4!1s0x880519721d04e425:0x810864e4979e1bed!8m2!3d43.0484696!4d-87.9187688)
+* https://medium.com/@danbarrett/north-6th-street-and-mckinley-avenue-milwaukee-wisconsin-7-15-to-7-28-p-m-part-1-89ad38a5532f
+* [Police are reviewing the use of force](https://www.jsonline.com/story/news/crime/2020/06/04/mpd-reviewing-use-force-arrest-during-6th-and-mckinley-standoff/3142673001/)
+* [Coverage of the lawsuit](https://www.jsonline.com/story/news/2020/07/01/cyclist-arrested-at-protest-files-500-000-claim-with-city/5351811002/)
+* [Location via Google Street View](https://www.google.com/maps/@43.0470717,-87.9188693,3a,75y,140.42h,99.57t/data=!3m6!1e1!3m4!1sQruIkc5jDDB5HaPqpjulkA!2e0!7i16384!8i8192)
+
+
+### Police use excessive force to disperse protesters | June 2nd
+
+Protesters had congregated near the intersection of North 6th Street and McKinley Avenue. Over the course of about an hour, police moved forward, dispersing the protesters using tear gas and rubber bullets, first pushing the protesters a block and a half then pushing them off of North 6th Street entirely. Video shows police shooting rubber bullets at protesters on a sidewalk and firing gas canisters, in large number, at a small group of peaceful protesters. (At least some tear gas was used, but it is unclear how much. Police say they also used smoke canisters.)
+
+Police claim they used force to disperse the crowd after rocks and glass were thrown at them. Police also said at the time that "Molotov cocktails" were thrown at them. They later walked that statement back to say that one Molotov Cocktail was thrown. They say it did not ignite. Media and protesters present say they saw water bottles being thrown, but not glass, rocks, or a Molotov cocktail. Video released by police clearly shows something being thrown at police, but doesn't show what it was. Police also produced a picture of a crushed water bottle and what looks like a water pump and implied it was the remnants of a Molotov cocktail.
+
+NOTE: this incident immediately follows WI-Milwaukee-2. Because the misconduct involved is substantially different, it has been given its own incident.
+
+tags: gas, protestor, rubber-bullets, tear-gas
```

Comment:
"gas" isn't a tag. Also add "shoot." Maybe we should add "smoke-canister" as well as a new tag

Reply:
I believe gas is listed on "possible_tags.md"

As for adding a smoke-canister tag: Probably! They seem to crop up regularly.

Reply:
You're right about the gas tag! I think that's good enough since smoke alone isn't really a big deal